SConstruct

```diff
@@ -430,13 +427,16 @@ def scons(): # pylint: disable=too-many-locals
     # also install to $PREFIX/lib to work with existing avocado test code
     daos_build.install(env, "lib/daos/", ['.build_vars.sh', '.build_vars.json'])
     env.Install("$PREFIX/lib64/daos", "VERSION")
-    env.Install("$PREFIX/lib64/daos", "API_VERSION")
 
     env.Install('$PREFIX/etc/bash_completion.d', ['utils/completion/daos.bash'])
     env.Install('$PREFIX/lib/daos/TESTING/ftest/util',
                 ['utils/sl/env_modules.py'])
     env.Install('$PREFIX/lib/daos/TESTING/ftest/',
                 ['ftest.sh'])
+    api_version = env.Command("%s/API_VERSION" % build_prefix,
+                              "%s/SConstruct" % build_prefix,
+                              "echo %s > $TARGET" % (API_VERSION))
+    env.Install("$PREFIX/lib64/daos", api_version)
```

Comment:
Is this writing the file using echo, then installing the resulting "command" as a binary?  What's the file mode in this case?

Reply:
It's not installing as a binary.  Install just copies files around.

jvolivie-desk1:[~/daos]$ ls install/lib64/daos/API_VERSION -la
-rw-r--r--. 1 jvolivie jvolivie 6 Feb  8 12:13 install/lib64/daos/API_VERSION

There probably is a better way to do it but I want to get the main problem resolved first (that there is more than one place to set the version).